packages/session/src/index.ts

```diff
@@ -60,7 +60,7 @@ export type { SessionStore };
 const DEFAULT_COOKIE_NAME = 'session';
 const DEFAULT_COOKIE_MAX_AGE = 86400000; // 1 day
 
-export function createSessionMiddleware(options: SessionOptions) {
+export function createSessionMiddleware(options: SessionOptions): RequestHandler {
```

Comment:
I feel like I've seen this change across a few PRs but it's been backed out before. What problem was this trying to solve? This returns the return value of `asyncHandler` so it should be totally fine to rely on inference here.

Reply:
This was resolving some type error, that I believe has since been merged in.